### ISSUE TITLE
Exception while loading plugin flexget.plugins.api.tvdb_lookup

### ISSUE THREAD

**Issue by Pezmc** [Flexget/Flexget#974]
Appears to be on [this line](https://github.com/Flexget/Flexget/blob/develop/flexget/api/__init__.py#L156) when loading `tvdb_lookup`, I updated to 1.2.495 from somewhere around 1.2.470 and see this failure on cron/execute since.

```
$ python -v
Python 2.7.11
$ flexget -V
1.2.495
You are on the latest release.
```

```
flexget --execute
2016-03-26 11:15 WARNING  manager                       Your locale declares ascii as the filesystem encoding. Any plugins reading filenames from disk will not work properly for filenames containing non-ascii characters. Make sure your locale env variables are set up correctly for the environment which is launching FlexGet.
2016-03-26 11:15 CRITICAL plugin                        Exception while loading plugin flexget.plugins.api.tvdb_lookup
2016-03-26 11:15 ERROR    plugin                        'super' object has no attribute 'response'
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/flexget/plugin.py", line 381, in _load_plugins_from_dirs
    __import__(module_name)
  File "/usr/local/lib/python2.7/site-packages/flexget/plugins/api/tvdb_lookup.py", line 5, in <module>
    from flexget.api import api, APIResource
  File "/usr/local/lib/python2.7/site-packages/flexget/api/__init__.py", line 320, in <module>
    loader.find_module(module_name).load_module(module_name)
  File "/usr/local/lib/python2.7/pkgutil.py", line 246, in load_module
    mod = imp.load_module(fullname, self.file, self.filename, self.etc)
  File "/usr/local/lib/python2.7/site-packages/flexget/api/authentication.py", line 79, in <module>
    class LoginAPI(APIResource):
  File "/usr/local/lib/python2.7/site-packages/flexget/api/authentication.py", line 81, in LoginAPI
    @api.response(401, 'Invalid username or password')
  File "/usr/local/lib/python2.7/site-packages/flexget/api/__init__.py", line 156, in response
    return super(Api, self).response(code_or_apierror, description, model=model)
AttributeError: 'super' object has no attribute 'response'
Traceback (most recent call last):
  File "/usr/local/bin/flexget", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/site-packages/flexget/__init__.py", line 37, in main
    manager.start()
  File "/usr/local/lib/python2.7/site-packages/flexget/manager.py", line 302, in start
    self.initialize()
  File "/usr/local/lib/python2.7/site-packages/flexget/manager.py", line 176, in initialize
    plugin.load_plugins(extra_dirs=[os.path.join(self.config_base, 'plugins')])
  File "/usr/local/lib/python2.7/site-packages/flexget/plugin.py", line 425, in load_plugins
    _load_plugins_from_dirs(extra_dirs)
  File "/usr/local/lib/python2.7/site-packages/flexget/plugin.py", line 381, in _load_plugins_from_dirs
    __import__(module_name)
  File "/usr/local/lib/python2.7/site-packages/flexget/plugins/api/tvdb_lookup.py", line 5, in <module>
    from flexget.api import api, APIResource
  File "/usr/local/lib/python2.7/site-packages/flexget/api/__init__.py", line 320, in <module>
    loader.find_module(module_name).load_module(module_name)
  File "/usr/local/lib/python2.7/pkgutil.py", line 246, in load_module
    mod = imp.load_module(fullname, self.file, self.filename, self.etc)
  File "/usr/local/lib/python2.7/site-packages/flexget/api/authentication.py", line 79, in <module>
    class LoginAPI(APIResource):
  File "/usr/local/lib/python2.7/site-packages/flexget/api/authentication.py", line 81, in LoginAPI
    @api.response(401, 'Invalid username or password')
  File "/usr/local/lib/python2.7/site-packages/flexget/api/__init__.py", line 156, in response
    return super(Api, self).response(code_or_apierror, description, model=model)
AttributeError: 'super' object has no attribute 'response'
```

**Comment by stevezau** [comment]
is this still happening on the latest version?

**Comment by Pezmc** [comment]
```
$ pip install --upgrade setuptools
Collecting setuptools
  Downloading setuptools-20.6.7-py2.py3-none-any.whl (508kB)
...snip...
Successfully installed setuptools-20.6.7
$ pip install --upgrade flexget
Collecting flexget
  Downloading FlexGet-1.2.501-py27-none-any.whl (2.2MB)
...snip...
Successfully installed apscheduler-3.1.0 flexget-1.2.501 funcsigs-0.4 pytvmaze-1.4.9
$ flexget -V
1.2.501
You are on the latest release.
$ flexget --execute
2016-04-04 09:27 VERBOSE  task_queue                    There are 2 tasks to execute. Shutdown will commence when they have completed.
2016-04-04 09:27 VERBOSE  details       movies imdb     Produced 10 entries.
```

Seems to have been fixed somewhere between `1.2.495` and `1.2.501`!
